plugins/plugin-chart-table/src/DataTable/DataTable.tsx

```diff
@@ -98,149 +136,142 @@ export default function DataTable<D extends object>({
             ) : null}
-          </Row>
+          </div>
         </div>
       ) : null}
-      <div style={{ height: tableHeight, overflow: 'auto' }}>
-        <table ref={tableRef} {...getTableProps({ className })}>
-          <thead>
-            {headerGroups.map(headerGroup => (
-              <tr key={headerGroup.id} {...headerGroup.getHeaderGroupProps()}>
-                {headerGroup.headers.map(column => {
-                  const headerProps = column.getHeaderProps();
-                  const sortByProps = column.getSortByToggleProps();
-                  const props = {
-                    ...headerProps,
-                    ...sortByProps,
-                    style: {
-                      ...headerProps.style,
-                      ...sortByProps.style,
-                    },
-                  };
-                  props.className = column.isSorted
-                    ? `${props.className} is-sorted`
-                    : props.className;
-
-                  let sortIcon = <FaSort />;
-                  if (column.isSorted) {
-                    sortIcon = column.isSortedDesc ? <FaSortDown /> : <FaSortUp />;
-                  }
-
-                  return (
-                    <th key={column.id} {...props}>
-                      {column.render('Header')}
-                      {sortIcon}
-                    </th>
-                  );
-                })}
-              </tr>
-            ))}
-          </thead>
-          <tbody {...getTableBodyProps()}>
-            {page && page.length > 0 ? (
-              page.map(row => {
-                prepareRow(row);
-                return (
-                  <tr key={row.id} {...row.getRowProps()}>
-                    {row.cells.map(cell => {
-                      const key = cell.column.id;
-                      const cellProps = cell.getCellProps() as TableCellProps & RenderHTMLCellProps;
-                      const { cellContent, ...restProps } = cellProps;
-                      if (cellProps.dangerouslySetInnerHTML) {
-                        return <td key={key} {...restProps} />;
-                      }
-                      // If cellProps renderes textContent already, then we don't have to
-                      // render `Cell`. This saves some time for large tables.
-                      return (
-                        <td key={key} {...restProps}>
-                          {cellContent || cell.render('Cell')}
-                        </td>
-                      );
-                    })}
-                  </tr>
-                );
-              })
-            ) : (
-              <tr>
-                <td className="dt-no-results" colSpan={columns.length}>
-                  {t(globalFilter ? 'No matching records found' : 'No records found')}
-                </td>
-              </tr>
-            )}
-          </tbody>
-        </table>
-      </div>
+      {wrapStickyTable ? wrapStickyTable(renderTable) : renderTable()}
```

Comment:
This is just for making it possible to skip applying `useSticky` hook (via a `sticky: boolean` option).